servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/RetryingHttpRequesterFilter.java

```diff
@@ -196,7 +196,7 @@ public void onComplete() {
                             .takeWhile(lbEvent ->
                                     // Don't complete until we get a LoadBalancerReadyEvent that is ready.
                                     !(lbEvent instanceof LoadBalancerReadyEvent) ||
-                                    !((LoadBalancerReadyEvent) lbEvent).isReady())
+                                        !((LoadBalancerReadyEvent) lbEvent).isReady())
```

Comment:
I didn't mean indentation 😄 
I was talking about logical operators: && vs ||:
```
!(lbEvent instanceof LoadBalancerReadyEvent &&
        ((LoadBalancerReadyEvent) lbEvent).isReady()))
```

Reply:
oOOoohh gotcha, sure I can do this too